dashboard/src/stores/auth.ts

```diff
@@ -132,6 +136,19 @@ export const useAuthStore = defineStore("auth", {
         this.config.abac.claimValuePrefix =
           env.VITE_OIDC_ABAC_CLAIM_VALUE_PREFIX.trim();
       }
+      if (env.VITE_OIDC_ABAC_USE_ROLES) {
+        this.config.abac.useRoles =
+          env.VITE_OIDC_ABAC_USE_ROLES.trim().toLowerCase() === "true";
+      }
+      if (this.config.abac.useRoles && env.VITE_OIDC_ABAC_ROLES_MAPPING) {
```

Comment:
Is having a separate "useRoles" variable necessary, or could we just treat an empty `roles mapping` as disabling role based authentication?

Reply:
I had my conflicts about this too. In the end I decided to be explicit, following the same pattern we have for the `enabled` settings for auth and ABAC.